app/code/Magento/CatalogMessageBroker/Model/Publisher/ProductPublisher.php

```diff
@@ -196,7 +196,7 @@ private function importProducts(
         $importProductRequest->setProducts($productsRequestData);
         $importProductRequest->setStore($storeCode);
 
-        $importResult = $this->catalogServer->importProducts($importProductRequest);
+        $importResult = $this->connector->getConnection()->importProducts($importProductRequest);
```

Comment:
since we use one MB for several services need to have the ability to choose service by service name (e.g. "catalog", "variants", "review", ...)